test/component/Tooltip/Tooltip.visibility.spec.tsx

```diff
@@ -723,12 +723,12 @@ describe('Tooltip visibility', () => {
     expect(container.querySelector('.recharts-tooltip-cursor')).toBeVisible();
 
     // Data should be displayed in the Tooltip payload
-    expect(tooltip.textContent).toBe('100stature : 100cmweight : 200kg');
+    expect(tooltip.textContent).toBe('stature : 100cmweight : 200kg');
```

Comment:
The title disappeared? Why?

Reply:
The title is not there in 2.x. Same with the storybook diff removing what is essentially the index from the tooltip title on the shared bar and radial charts. Scatter never had a title